hs-bindgen-libclang/cbits/Makefile

```diff
@@ -6,8 +6,8 @@ all: clang_wrappers.o doxygen_wrappers.o
 clang_wrappers.o: clang_wrappers.c clang_wrappers.h
 	gcc -c -Wall clang_wrappers.c -I/usr/lib/llvm-14/include
 
-doxygen_wrappers.o: doxygen_wrappers.c doxygen_wrappers.h
-	gcc -c -Wall doxygen_wrappers.c -I/usr/lib/llvm-14/include
+doxygen_wrappers.o: doxygen_wrappers.h
+	gcc -c -Wall doxygen_wrappers.h -I/usr/lib/llvm-14/include
```

Comment:
This doesn't look right.

Reply:
Yeah, it will, but the only purpose of this `Makefile` is for me to make it a bit easier to get all compiler warnings.